fuzz/Cargo.toml

```diff
@@ -37,3 +37,9 @@ name = "inflate"
 path = "fuzz_targets/inflate.rs"
 test = false
 doc = false
+
+[[bin]]
+name = "inflate_miniz"
+path = "fuzz_targets/inflate_miniz.rs"
```

Comment:
I'm slightly torn on whether this is the right name given that it is actually testing the decompression half of this crate. But "deflate_miniz" also seems wrong, so...

Reply:
`deflate_miniz` would be the inverse - anything encoded by this crate can be decoded by miniz_oxide; which is also a fuzz target I should probably write, now that I think of it.

Reply:
Oh nevermind that already exists. It uses flate2 but that's just a wrapper around several backends, and miniz_oxide is the default one, which is used here.